commit df77b4d9c15823519165f6e9a206b2c40ac68389
Author: mjw <mjw@a5019735-40e9-0310-863c-91ae7b9d1cf9>
Date:   Mon Mar 10 14:45:32 2014 +0000

    Enable sys_getpriority and sys_setpriority for ppc64.
    
    Bug 331830 - ppc64: WARNING: unhandled syscall: 96/97
    (Miroslav Franc, mfranc@redhat.com)
    
    git-svn-id: svn://svn.valgrind.org/valgrind/trunk@13861 a5019735-40e9-0310-863c-91ae7b9d1cf9

--- a/coregrind/m_syswrap/syswrap-ppc64-linux.c
+++ b/coregrind/m_syswrap/syswrap-ppc64-linux.c
@@ -792,8 +792,8 @@ static SyscallTableEntry syscall_table[] = {
    GENX_(__NR_fchmod,            sys_fchmod),             //  94
    
    GENX_(__NR_fchown,            sys_fchown),             //  95
-// _____(__NR_getpriority,       sys_getpriority),        //  96
-// _____(__NR_setpriority,       sys_setpriority),        //  97
+   GENX_(__NR_getpriority,       sys_getpriority),        //  96
+   GENX_(__NR_setpriority,       sys_setpriority),        //  97
 // _____(__NR_profil,            sys_profil),             //  98
    GENXY(__NR_statfs,            sys_statfs),             //  99
 
